tests/integration/security/ca_custom_root/main_test.go

```diff
@@ -271,9 +321,50 @@ func setupConfig(_ resource.Context, cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
-	cfg.ControlPlaneValues = `
+
+	// Add root certificate to list of trusted anchors
+	// TODO: Read certificate from directory
+	// script := path.Join(env.IstioSrc, "samples/certs", "root-cert-2.pem")
+	// rootPEM, err := loadCert(script)
+	// if err != nil {
+	//	return
+	//}
+
+	cfgYaml := `
 values:
   meshConfig:
     trustDomainAliases: [some-other, trust-domain-foo]
+    caCertificates:
+    - pem: |
+        -----BEGIN CERTIFICATE-----
```

Comment:
can we get this from whatever file its from?

Reply:
working on it..not able to find a straightforward way to add multiline strings to yaml in golang

Reply:
oh yeah the indent is annoying - i remember that. You could use `tmpl.MustEvaluate` then `{{.cert | indent 6}}` or similar though

Reply:
cool, will try this